velox/vector/tests/VectorTestBase.h

```diff
@@ -148,6 +148,14 @@ class VectorTestBase {
     return vectorMaker_.flatVectorNullable(data, type);
   }
 
+  FlatVectorPtr<int64_t> makeShortDecimalFaltVector(
```

Comment:
typo: `makeShortDecimalFlatVector`.
This can be `makeDecimalFlatVector`. We are passing precision, scale, which are sufficient.